src/NHibernate/Tuple/Entity/DynamicMapEntityTuplizer.cs

```diff
@@ -13,7 +13,7 @@ public class DynamicMapEntityTuplizer : AbstractEntityTuplizer
 	{
 		private static readonly INHibernateLogger log = NHibernateLogger.For(typeof(PocoEntityTuplizer));
 
-		internal DynamicMapEntityTuplizer(EntityMetamodel entityMetamodel, PersistentClass mappingInfo)
+		public DynamicMapEntityTuplizer(EntityMetamodel entityMetamodel, PersistentClass mappingInfo)
```

Comment:
All other tuplizers, including the dynamic component one, have a public constructor. Let's fix this inconsistency.